velero-plugin-for-aws/volume_snapshotter.go

```diff
@@ -62,69 +63,64 @@ func (b *VolumeSnapshotter) Init(config map[string]string) error {
 	region := config[regionKey]
 	credentialProfile := config[credentialProfileKey]
 	credentialsFile := config[credentialsFileKey]
-	enableSharedConfig := config[enableSharedConfigKey]
+	//	enableSharedConfig := config[enableSharedConfigKey]
```

Comment:
This line may not be needed.

Reply:
Thanks I'll make the change in a follow-up PR.